lebedigital/demonstrator_scripts/computation_GWP_mix.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+from lebedigital.unit_registry import ureg
+
+def computation_GWP_mix(constituents):
+    """
+    This function computes the global warming potential for a square meter of cement mix
+
+    Parameters
+    ----------
+    constituents : dict
+        the dictonary expects e.g.:
+        {'cement' : {'content': 5 * ureg('kg/m^3'), 'GWP': 2 * ureg('kg_CO2_eq/kg')},
+         'aggregates': {'content': 10 * ureg('kg/m^3'), 'GWP': 1 * ureg('kg_CO2_eq/kg')}}
+
+    Returns
+    -------
+    gwp_mix : float / pint unit, will be in 'kg_CO2_eq/m^3'
```

Comment:
I see your point, but we are not actually measuring the mass of CO2. I think this makes it a bit more clear, otherwise the result will look like a density.